assign_nodes.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+# Read in the user example file and output as a pandas dataframe
+def read_user_input(user_example_file):
+	examples = pd.read_csv(user_example_file)
+	return(examples)
+
+# Map the nodes to a dictionary
+def map_nodes(examples):
+	nodes = pd.melt(examples)
+	return(nodes)
+
+# Search through graph to find nodes based on input features
+def findNode(node_name, nodes, ontology = ""):
```

Comment:
function names should be word1_word2 format

Reply:
test